pubspec.yaml

```diff
@@ -19,9 +19,12 @@ environment:
   sdk: '>=2.19.0-0 <4.0.0'
   flutter: ">=3.7.0-0"
 
-# dependency_overrides:
-#   vector_graphics:
-#     path: ../vector_graphics/packages/vector_graphics
+dependency_overrides:
```

Comment:
I'll publish a new VG version sometime soon and we can back this out.. Otherwise this patch LGTM.

Reply:
Hey, just wanted to check in again and ask if this PR can be merged soon. I would like to remove the depedency_override 😄

Reply:
I've manually updated this and will merge as long as tests are passing.